__tests-pacts__/serlo.org/threads.ts

```diff
@@ -0,0 +1,24 @@
+import { Matchers } from '@pact-foundation/pact'
+import fetch from 'node-fetch'
+
+import { article } from '../../__fixtures__/uuid'
+import { ThreadsPayload } from '../../src/graphql/schema/threads'
+import { addJsonInteraction } from '../__utils__'
+
+test('Threads', async () => {
+  // This is a noop test that just adds the interaction to the contract
+  const threads: ThreadsPayload = {
+    threadIds: [10, 20, 30],
+  }
+  await addJsonInteraction({
+    name: `fetch data of thread ${threads.threadIds[0]}`,
+    given: `threads ${threads.threadIds[0]} is of type thread`,
+    path: `/api/uuid/${threads.threadIds[0]}`,
+    body: {
+      threadIds: Matchers.eachLike(Matchers.integer(1)),
+    },
+  })
```

Comment:
```suggestion
  await addJsonInteraction({
    name: `fetch first comment ids of all threads for an article`,
    given: `article ${article.id} has threads`,
    path: `/api/threads/${article.id}`,
    body: {
      threadIds: Matchers.eachLike(Matchers.integer(1)),
    },
  })
```

When you call `yarn pacts` there shall not be an error, when everything is right.